workflow/schemas/config.schema.yaml

```diff
@@ -233,15 +233,28 @@ properties:
               type: string
       vep:
         properties:
-          params:
-            type: string
-          plugins:
-            type: array
-            items:
-              type: string
-        required:
-          - params
-          - plugins
+          candidate_calls:
+            properties:
+              params:
+                type: string
+              plugins:
+                type: array
+                items:
+                  type: string
+            required:
+              - params
+              - plugins
+          final_calls:
+            properties:
+              params:
+                type: string
+              plugins:
+                type: array
+                items:
+                  type: string
+            required:
+              - params
+              - plugins
```

Comment:
I think this now lacks a "required"-section for the two cases.

Reply:
It does. Added them just now. Thanks for catching this!